### ISSUE TITLE
No query results, Dataview plugin said to be not installed when it is

### ISSUE THREAD

**Issue by FitzFrobozz** [sytone/obsidian-queryallthethings#10]
I have dataview installed, enabled, and working (i.e. I can run dataview queries just fine.) When I enable queryallthethings and try to run the example query listed in the README > Getting Started, or the one in Documentation > Your First query, no results are returned. When I disable queryallthethings and then re-enable it, I get this warning: "Dataview plugin is not installed. Dataview backed tables will be empty." Is there another prerequisite that I overlooked?

**Comment by sytone** [comment]
No. It should work if dataview is enabled and using the dataview tables. 

Are you trying to run this query?

```qatt
query: |
  SELECT TOP 5 *
  FROM obsidian_markdown_notes
  ORDER BY stat->mtime DESC
template: |
  {{#each result}}
   - [[{{path}}\|{{basename}}]]
  {{/each}}
```

**Comment by FitzFrobozz** [comment]
That's the one, yeah.

**Comment by sytone** [comment]
Issue has been resolve. The new handlebars helpers caused a conflict that was not apparent on initial local testing. This has been reverted. I also detected a async race condition in the checks for dataview and customjs. These have also been resolved. Initial fixes were in 0.8.9 and the CustomJS fix was in 0.8.10

**Comment by sytone** [comment]
Confirmed working via Discord chat in plugin-general. Closing up issue.
